Find a functionally equivalent code:android.widget.DatePicker.getMinDate:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the minimal date supported by this {@link DatePicker} in
 * milliseconds since January 1, 1970 00:00:00 in
 * {@link TimeZone#getDefault()} time zone.
 * <p>
 * Note: The default minimal date is 01/01/1900.
 * <p>
 *
 * @return The minimal supported date.
 */

@@ -1,3 +1,3 @@
 {
-    return mCalendarView.getMinDate();
+    return mDelegate.getMinDate().getTimeInMillis();
 }

